tooling/cli.rs/metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-  "cli.js": {
+  "jsCli": {
```

Comment:
why did this get changed?

Reply:
this is what caused the panic in the first place,
we have this struct in rust and we use serde to deserialize from json into it.
```rust
#[derive(Deserialize)]
#[serde(rename_all = "camelCase")]
struct VersionMetadata {
  js_cli: JsCliVersionMetadata,
}
```
there is a miss-match between the json and the strcut and the easier way was to change the json.

Reply:
actually I don't need to change the `metadata.json`, will push a fix on the rust side soon.